config.json

```diff
@@ -629,7 +629,13 @@
         "name": "Nucleotide Count",
         "uuid": "be6f3ab3-1593-4c2d-8a35-5f39c1d5c91f",
         "practices": [],
-        "prerequisites": [],
+        "prerequisites": [
+          "classes",
+          "array-transformation",
+          "objects",
+          "strings",
+          "template-strings"
+        ],
```

Comment:
This exercise is a tough one to define. The proof file solution is not very good imo and the community solutions are full of awful solutions as well.

I digged around a bit and I would use something like this as guidance for the prerequisites: https://exercism.org/tracks/javascript/exercises/nucleotide-count/solutions/shybyte (with the change that instead of the spread operator (`...`) there could be a simple `split`).

That would leave us with classes, objects, strings, array-loops, conditionals and errors as prerequisites. What do you think about that?

Reply:
Agreed, I've done something similar myself: https://exercism.org/tracks/javascript/exercises/nucleotide-count/solutions/TomPradat

I'm wondering about conditionals though, it seems to me that 4 prerequisites are long enough. What's more, conditionals should be known before learning about objects. What do you think?